app/src/main/res/values/themes.xml

```diff
@@ -6,4 +6,22 @@
     </style>
 
     <style name="Theme.OneLogin" parent="Base.Theme.OneLogin" />
+
+    <style name="Theme.App.OneLogin" parent="Theme.SplashScreen">
+        <!-- Set the splash screen background, animated icon, and animation
+        duration. -->
+        <item name="windowSplashScreenBackground">@color/icon_background</item>
+
+        <!-- Use windowSplashScreenAnimatedIcon to add a drawable or an animated
+             drawable. One of these is required. -->
+        <item name="windowSplashScreenAnimatedIcon">@drawable/ic_launcher_foreground</item>
+
+        <!-- Required for animated icons. -->
+<!--        <item name="windowSplashScreenAnimationDuration">200</item>-->
```

Comment:
Adding this line back in, and removing the viewModel delay might be the best course of action for now